pkg/performanceprofile/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -22,20 +23,18 @@ const expectedMatchSelector = `
 `
 
 var (
-	cmdlineCPUsPartitioning                 = regexp.MustCompile(`\s*cmdline_cpu_part=\+\s*nohz=on\s+rcu_nocbs=\${isolated_cores}\s+tuned.non_isolcpus=\${not_isolated_cpumask}\s+systemd.cpu_affinity=\${not_isolated_cores_expanded}\s+intel_iommu=on\s+iommu=pt\s*`)
-	cmdlineWithStaticIsolation              = regexp.MustCompile(`\s*cmdline_isolation=\+\s*isolcpus=managed_irq,\${isolated_cores}\s*`)
-	cmdlineWithoutStaticIsolation           = regexp.MustCompile(`\s*cmdline_isolation=\+\s*isolcpus=domain,managed_irq,\${isolated_cores}\s*`)
-	cmdlineWithRealtime                     = regexp.MustCompile(`\s*cmdline_realtime=\+\s*nohz_full=\${isolated_cores}\s+tsc=nowatchdog\s+nosoftlockup\s+nmi_watchdog=0\s+mce=off\s+skew_tick=1\s*`)
-	cmdlineHighPowerConsumption             = regexp.MustCompile(`\s*cmdline_power_performance=\+\s*processor.max_cstate=1\s+intel_idle.max_cstate=0\s+intel_pstate=disable\s*`)
-	cmdlineHighPowerConsumptionWithRealtime = regexp.MustCompile(`\s*cmdline_idle_poll=\+\s*idle=poll\s*`)
-	cmdlineHugepages                        = regexp.MustCompile(`\s*cmdline_hugepages=\+\s*default_hugepagesz=1G\s+hugepagesz=1G\s+hugepages=4\s*`)
-	cmdlineAdditionalArg                    = regexp.MustCompile(`\s*cmdline_additionalArg=\+\s*test1=val1\s+test2=val2\s*`)
-	cmdlineDummy2MHugePages                 = regexp.MustCompile(`\s*cmdline_hugepages=\+\s*default_hugepagesz=1G\s+hugepagesz=1G\s+hugepages=4\s+hugepagesz=2M\s+hugepages=0\s*`)
-	cmdlineMultipleHugePages                = regexp.MustCompile(`\s*cmdline_hugepages=\+\s*default_hugepagesz=1G\s+hugepagesz=1G\s+hugepages=4\s+hugepagesz=2M\s+hugepages=128\s*`)
+	cmdlineCPUsPartitioning       = "+nohz=on rcu_nocbs=${isolated_cores} tuned.non_isolcpus=${not_isolated_cpumask} systemd.cpu_affinity=${not_isolated_cores_expanded} intel_iommu=on iommu=pt"
+	cmdlineWithStaticIsolation    = "+isolcpus=domain,managed_irq,${isolated_cores}"
+	cmdlineWithoutStaticIsolation = "+isolcpus=managed_irq,${isolated_cores}"
+	cmdlineRealtime               = "+nohz_full=${isolated_cores} tsc=nowatchdog nosoftlockup nmi_watchdog=0 mce=off skew_tick=1 rcutree.kthread_prio=11"
+	cmdlineHighPowerConsumption   = "+processor.max_cstate=1 intel_idle.max_cstate=0 intel_pstate=disable"
+	cmdlineIdlePoll               = "+idle=poll"
+	cmdlineHugepages              = "+ default_hugepagesz=1G   hugepagesz=1G hugepages=4"
+	cmdlineAdditionalArgs         = "+ audit=0 processor.max_cstate=1 idle=poll intel_idle.max_cstate=0"
```

Comment:
I see we have moved away from `"test1=val1", "test2=val2"` to more realistic examples for kernel params.  IIUC, we are providing a set of  random kernel arguments here and later checking against them. Is my understanding here correct? If that is the case, lets add a comment here that this is an example of how additional kernel arguments could look like and these arguments have been selected at random for demonstration purposes and have no other significance.

Reply:
Done, right. I had moved these test1=val1 to the testing.go profile and add a more "real ones". To have everything in the same file and tidier. 
Add comment before the test.